task_processing/plugins/mesos/execution_framework.py

```diff
@@ -40,19 +40,21 @@ def __init__(
         self.name = name
         # wait this long for a task to launch.
         self.task_staging_timeout_s = task_staging_timeout_s
-        self.framework_info = self.build_framework_info()
         self.pool = pool
         self.translator = translator
         self.slave_blacklist_timeout_s = slave_blacklist_timeout_s
         self.offer_backoff = offer_backoff
         self.task_retries = task_retries
 
+        # TODO: why does this need to be root, can it be "mesos plz figure out"
+        self.framework_info = Dict(
+            user='root', name=self.name, checkpoint=True)
```

Comment:
nit: Can we have each argument on a separate line? I find it easier to read. But, nbd.

self.framework_info = Dict(
      user='root', 
      name=self.name, 
      checkpoint=True
)